packages/Alert/src/index.tsx

```diff
@@ -1,26 +1,35 @@
 import Button from "@blaze-react/button";
+import WithUtils from "@blaze-react/utils";
 import React, { Fragment, FunctionComponent, useState } from "react";
 interface IAlertProps {
   icon?: string;
   type?: string;
   close?: boolean;
-  children?: JSX.Element | string | {} | [];
+  utils: {
+    classNames: (...args: any) => string;
+  };
+  children?: JSX.Element | string;
 }
 
 const Alert: FunctionComponent<IAlertProps> = ({
   children,
   close,
   icon,
   type,
+  utils: { classNames },
   ...attrs
 }): JSX.Element => {
-  const [offModal, setModalOff] = useState(false);
-  const assignType = type && `alert--${type}`;
-  const isDismissable = close && "alert--dismissable";
-  const withIcon = icon && "alert--icon";
+  const [offModal, setModalOff] = useState<boolean>(false);
+
+  const iconClassName: string = classNames({ "alert--icon": !!icon });
+  const typeClassName: string = classNames({ [`alert--${type}`]: !!type });
+  const dismissableClassName: string = classNames({
+    "alert--dismissable": close
+  });
```

Comment:
this could be 
```
const alertClassname: string = classNames('alert', {
  "alert--icon": !!icon ,
  [`alert--${type}`]: !!type,
  "alert--dismissable": close
})

```

In the div you will have

```
<div className={alertClassname} {...attrs} >	
```

Reply:
The first agument of classnames is the default class in this case alert